main.lua

```diff
@@ -231,6 +262,27 @@ function renderFieldOutline()
   dottedLine(0.5 * VIRTUAL_WIDTH, BOARD_BORDER_TOP, 0.5 * VIRTUAL_WIDTH, VIRTUAL_HEIGHT, 5, 3)
 end
 
+function logBallOwnership()
+  love.graphics.setColor(getColor('red'))
+  love.graphics.printf('Ball ownership: ' ..BALL_POSSESION, 10, BORDER_LEFT_RIGHT + 10, 300)
+end
+
+function logballPosition()
```

Comment:
I have just noticed that I've made a typo here! I will fix it after we will discuss all the changes